modules/core/server/services/influx.server.service.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var path = require('path'),
+    influx = require('influx'),
+    config = require(path.resolve('./config/config'));
+
+/**
+ * Get InfluxDB Client
+ */
+exports.getClient = function(callback) {
+  if (!config.influxdb ||
+      !config.influxdb.enabled ||
+      !config.influxdb.options ||
+      !config.influxdb.options.host ||
+      !config.influxdb.options.database) {
+    return callback(new Error('No InfluxDB configured.'));
+  }
+  callback(null, influx(config.influxdb.options));
+};
+
+/**
+ * Write point to InfluxDB
+ *
+ * `values` can be either an object or a single value. For the latter the columname is set to value.
+ * You can set the time by passing an object propety called time. The time an be either an integer value or a Date object.
+ * When providing a single value, don't forget to adjust the time precision accordingly. The default value is `ms`.
```

Comment:
Great to see comments in the code. This comment is a little complicated to understand what it means. We could rewrite it something like:

> `values` can be either a single value or an object. If it's a single value, the column name will be the string `value`.
> You can set the time by passing a property called `time` (only if you pass an object obviously!). The time can either be an integer (default ms) or a date object.

I'm not actually sure what the last line means, about providing a single value and time precision. Maybe we can expand that to make it clearer.